app/views/smart_proxies/_tftp.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+  <div class="col-md-12">
+    <div class="row">
+      <h4><%= _('TFTP Details') %></h4>
+    </div>
+    <%= render_plugin_version(@smart_proxy, feature) %>
+    <div class="row">
+      <div class="col-md-4">
+        <%= _('TFTP server') %>
+      </div>
+      <div class="col-md-8">
+        <span class="proxy-version" data-url="<%= tftp_server_smart_proxy_path(@smart_proxy) %>">
+          <%= image_tag '/assets/spinner.gif' %>
+        </span>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
I know it can be little bit early and there is a separate ticket, but in this approach, how plugins are supposed to add new entries here? The Services page on Smart Proxy show screen looks like ideal candidate for our widgeting subsystem! Each plugin can provide it's own widget. Plus we get nice and customizable user interface.

This actually applies also to the 1st tab (Overview).